docs/edit/runbook.md

```diff
@@ -8,8 +8,6 @@ Here's what you need to do!
 
 #### Summary
 
-This can be **easily fixed** in ~5 minutes
```

Comment:
I think this is a bit misleading and can lead to frustration. I agree that active work is small but having to open two PRs in two different repos (which you may have never work with) and waiting for the review of a team is definitely not 5 minutes 😄

Reply:
I wrote the doc here so may be biased, but part of the reason to include this line was to encourage folks to fix it rather than marking the test as `bug`/`missing_feature`

Considering the above, wdyt about changing this to instead be something like below?

```
This can be fixed with **~5 minutes of effort**
```

Reply:
yeah I like that better 👍

Reply:
https://github.com/DataDog/system-tests/pull/4098/commits/39d50d8bf87228c137a90e98768fe17c0338d4c3